teamB/junhyungcho/models/postDao.js

```diff
@@ -0,0 +1,107 @@
+//데이터베이스와 연결, initialize 객체를 호출
+
+const { DataSource } = require('typeorm');
+const database = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+});
+
+database
+    .initialize()
+    .then(() => {
+        console.log("Post data Source has been initialized!");
+    })
+    .catch((err) => {
+        console.error('Error occurred during Data Source initialization', err);
+        database.destroy();
+    });
+
+const createPost = async (title, description, cover_image, users_id) => {
+    try {
+        const post = await database.query(
+            `INSERT INTO posts(
+                title,
+                description, 
+                cover_image, 
+                users_id
+                ) 
+                VALUES (?, ?, ?, ?);
+            `,
+            [ title, description, cover_image, users_id ]
+        );
+
+        return post;
+    }
+    catch (err) {
+        const error = new Error('INVALID_DATA_INPUT');
+        error.statusCode = 500;
+        throw error;
+    }
+};
+
+const getAllPost = async () => {
+    try {
+        return await database.query(
+        `SELECT * FROM posts`,
+        );
+    }
+    catch (err) {
+        const error = new Error('INVALID_DATA_INPUT');
+        error.statusCode = 500;
+        throw error;
+    };
+};
+
+const editPost = async ( postId, title, description, cover_image, users_id ) => {
```

Comment:
edit이라는 용어보다 update나 modify라는 용어를 더 많이 사용합니다.

```suggestion
const updatePost = async ( postId, title, description, cover_image, users_id ) => {
```